docs-2.0/7.data-security/3.manage-snapshot.md

```diff
@@ -0,0 +1,93 @@
+# Manage snapshots
+
+Nebula Graph supports using snapshots to backup and restore data.
+
+## Prerequisites
+
+If [authentication](1.authentication/1.authentication.md) is enabled, only the GOD-role user can use the snapshot function. For more information about roles, see [Roles and privileges](1.authentication/3.role-list.md).
+
+## Precautions
+
+* To prevent data loss, create a snapshot as soon as the system structure changes, for example, after operations such as `ADD HOST`, `DROP HOST`, `CREATE SPACE`, `DROP SPACE`, and `BALANCE` are performed.
```

Comment:
structure changes, -> structure changes.
comma -> period?

Reply:
Should be comma here.